### PR TITLE
chore: adding eslint and prettier

### PR DESCRIPTION
**Pull request by srikanthv02**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -141,5 +143,16 @@
       "@commitlint/config-conventional"
     ]
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
```
Comment (WilliamChelman) [inline package.json:148]: Is this needed seeing the content of `commit-msg`? 🤔

Comment (WilliamChelman) [reply package.json:148]: Or could the change to commit-msg be removed? I feel like one or the other is superfluous (and running it on commit-msg feels a bit wrong compared to pre-commit 🤔 )

Comment (srikanthv02) [reply package.json]: does not work without `commit-msg` :( . So keeping it there

**Comment by WilliamChelman** [comment]
Closing but integrated recently. It was just easier to recreate than to fix the conflicts 😛

**Pull request closed without merging**